docs/.vitepress/config.mts

```diff
@@ -39,9 +39,9 @@ if (inProd) {
   )
 
   // CookieYes
-  // head.push(
-  //   ['script', { id: 'cookieyes', src: 'https://cdn-cookieyes.com/client_data/43ed933a5c31287e10124caf/script.js' }]
-  // )
+  head.push(
+    ['script', { id: 'cookieyes', type: 'text/javascript', src: 'https://cdn-cookieyes.com/client_data/43ed933a5c31287e10124caf/script.js' }]
```

Comment:
Should this use an `async` attribute?

Reply:
not really... their site says to load it sync, i assume that it must block the page so it can prevent the cookies from loading while this loads.